yaksh/models.py

```diff
@@ -151,21 +151,21 @@ class Course(models.Model):
 
     objects = CourseManager()
 
+    def _create_duplicate_instance(self, creator, course_name=None):
+        new_course = self
+        new_course.id = None
+        new_course.name = course_name if course_name else self.name
```

Comment:
new_course.name is already self.name because 1st line in the method is ```new_course = self```
Actually no harm.